### ISSUE TITLE
polymer-js required in this case

### ISSUE THREAD

**Issue by nippur72** [greenify/polymer-js#1]
It's not a issue, I only wanted to mention [a specific case](https://github.com/Polymer/polymer/issues/2101) where using HTML imports to load Polymer is not possible (or at least not without "workarounds"), because of an issue in webcomponents polyfill. 

In such case, `polymer-js` is very handy.

**Comment by wilzbach** [comment]
I totally agree - in my humbled opinion we should get the ability to require polymer as JavaScript package into the official upstream  :)

**Comment by ajnsit** [comment]
Polymer is pure JS code, there should be no reason it's not available as a js file. Thanks for this workaround!
